src/libraries/System.Runtime/tests/System/IntPtrTests.cs

```diff
@@ -844,51 +844,135 @@ public static IEnumerable<object[]> Parse_ValidWithOffsetCount_TestData()
+            yield return new object[] { "100", 0, 1, NumberStyles.Currency, emptyCurrencyFormat, (IntPtr)1 };
 
             // If CurrencySymbol and Negative are the same, NegativeSign is preferred
             NumberFormatInfo sameCurrencyNegativeSignFormat = new NumberFormatInfo()
             {
                 NegativeSign = "|",
                 CurrencySymbol = "|"
             };
-            yield return new object[] { "1000", (IntPtr)1, (IntPtr)3, NumberStyles.AllowCurrencySymbol | NumberStyles.AllowLeadingSign, sameCurrencyNegativeSignFormat, (IntPtr)0 };
-            yield return new object[] { "|1000", (IntPtr)0, (IntPtr)2, NumberStyles.AllowCurrencySymbol | NumberStyles.AllowLeadingSign, sameCurrencyNegativeSignFormat, -1 };
+            yield return new object[] { "1000", 1, 3, NumberStyles.AllowCurrencySymbol | NumberStyles.AllowLeadingSign, sameCurrencyNegativeSignFormat, (IntPtr)0 };
+            yield return new object[] { "|1000", 0, 2, NumberStyles.AllowCurrencySymbol | NumberStyles.AllowLeadingSign, sameCurrencyNegativeSignFormat, (IntPtr)(-1) };
 
             // Any
-            yield return new object[] { "123", (IntPtr)0, (IntPtr)2, NumberStyles.Any, null, (IntPtr)12 };
+            yield return new object[] { "123", 0, 2, NumberStyles.Any, null, (IntPtr)12 };
 
             // AllowLeadingSign
-            yield return new object[] { "-2147483648", (IntPtr)0, (IntPtr)10, NumberStyles.AllowLeadingSign, null, -214748364 };
+            yield return new object[] { "-2147483648", 0, 10, NumberStyles.AllowLeadingSign, null, (IntPtr)(-214748364) };
 
             // AllowTrailingSign
-            yield return new object[] { "123-", (IntPtr)0, (IntPtr)3, NumberStyles.AllowTrailingSign, null, (IntPtr)123 };
+            yield return new object[] { "123-", 0, 3, NumberStyles.AllowTrailingSign, null, (IntPtr)123 };
 
             // AllowExponent
-            yield return new object[] { "1E2", (IntPtr)0, (IntPtr)1, NumberStyles.AllowExponent, null, (IntPtr)1 };
-            yield return new object[] { "1E+2", (IntPtr)3, (IntPtr)1, NumberStyles.AllowExponent, null, (IntPtr)2 };
-            yield return new object[] { "(1E2)", (IntPtr)1, (IntPtr)3, NumberStyles.AllowExponent | NumberStyles.AllowParentheses, null, (IntPtr)1E2 };
-            yield return new object[] { "-1E2", (IntPtr)1, (IntPtr)3, NumberStyles.AllowExponent | NumberStyles.AllowLeadingSign, null, (IntPtr)1E2 };
+            yield return new object[] { "1E2", 0, 1, NumberStyles.AllowExponent, null, (IntPtr)1 };
+            yield return new object[] { "1E+2", 3, 1, NumberStyles.AllowExponent, null, (IntPtr)2 };
+            yield return new object[] { "(1E2)", 1, 3, NumberStyles.AllowExponent | NumberStyles.AllowParentheses, null, (IntPtr)1E2 };
+            yield return new object[] { "-1E2", 1, 3, NumberStyles.AllowExponent | NumberStyles.AllowLeadingSign, null, (IntPtr)1E2 };
+        }
+
+        [Theory]
```

Comment:
Nit: the string-based routines are above the test data.  Can you please move these span-based tests to co-locate them with the corresponding string-based ones?

Reply:
Like this? 
before:
![image](https://user-images.githubusercontent.com/14993992/106288268-c8db0700-6282-11eb-89cb-d165c6c2b967.png)
after
![image](https://user-images.githubusercontent.com/14993992/106288330-dbedd700-6282-11eb-873a-65d6a5644a3b.png)
I noticed that methods are also organized like this in other tests `UInt32 UInt16 UInt64 Int16 etc`, do I need to change them too?
![image](https://user-images.githubusercontent.com/14993992/106288797-73532a00-6283-11eb-8fa9-10ff38a82c7d.png)

(File: [Int32Tests.cs](https://github.com/Cyl18/runtime/blob/ad7a50ea4e3968a4582b88367de412ecdc019578/src/libraries/System.Runtime/tests/System/Int32Tests.cs))

Reply:
![image](https://user-images.githubusercontent.com/14993992/106290175-240df900-6285-11eb-950c-0ffb04202079.png)
I think they are already corresponding the string-based ones?

Reply:
![image](https://user-images.githubusercontent.com/14993992/106290415-6800fe00-6285-11eb-9271-b1643530d634.png)
I think they are already corresponding the string-based ones?

Reply:
I talked with @stephentoub briefly offline and he was suggesting that the new span-based `Parse`/`Format` tests and test data be moved next to the string-based variants of the same

None of the other primitive types (like `Int32`) currently match this and so we decided not to block, but it seems like a good low priority/up-for-grabs issue if someone else feels like the consistency would help.